### PR TITLE
Fixed an error with the config in the master branch

### PR DESCRIPTION
**Pull request by midcoreboot**
Fixed a fuck up in the master branch of the database team not checking the config correctly before commiting, with this hotfix one can choose to use the local database in the env file.

### CODE REVIEW

**Review by joelAndreasson — APPROVED** [review]
I don't know enough about the config to give a proper review, so I am just trusting you on this one 👍🏻

**Review by LasseLejon — APPROVED** [review]

File: backend-app/src/config/validations.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ const joi = require("joi");
 const validationSchema = joi.object()
     .keys({
         PORT: joi.number(),
-        USE_LOCAL_DATABASE: joi.binary().integer().min(0).max(1),
+        USE_LOCAL_DATABASE: joi.number().required(),
```
Comment (h-khalaf) [inline backend-app/src/config/validations.js:6]: I assume the allowed values should be 0 and 1, in that case, you should add validation.

**Review by h-khalaf — CHANGES_REQUESTED** [review]

File: backend-app/src/config/validations.js (hunk 1 above)
Comment (midcoreboot) [reply backend-app/src/config/validations.js:6]: I understand this is a good practice, but when building this on the production server the integer function wouldn't work. This is why we chose to use the same way of checking as the OCPP team did.

Comment (h-khalaf) [reply backend-app/src/config/validations.js:6]: yeah its not necessary, it was just a recommendation.

**Review by h-khalaf — APPROVED** [review]

**Pull request merged**